inventory/group_vars/all.yml

```diff
@@ -231,6 +231,11 @@ nginx_acme_challenge_config: |+
     root /etc/letsencrypt/webrootauth;
   }
 
+nginx_valid_methods: |+
+  limit_except GET POST PUT DELETE OPTIONS {
```

Comment:
What about PATCH and HEAD? they are also valid and one would expect them to work.

Reply:
`HEAD` is included with `GET` in the nginx `limit_except` directive :+1: 

I forgot `PATCH` because we don't actually use it. I think it might get more use by Rails 4.2 though, I'll add it.